src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -1191,6 +1190,34 @@ int MachCallNativeNode::ret_addr_offset() {
   return -1;
 }
 
+// RVC: With RVC a call may get 2-byte aligned.
+//   The offset encoding in jal ranges bits [12, 31], which could span the cache line.
+//   Patching this unaligned address will make the write operation not atomic.
+//   Other threads may be running the same piece of code at full speed, causing concurrency issues.
+//   So we must ensure that it does not span a cache line so that it can be patched.
+int CallStaticJavaDirectNode::compute_padding(int current_offset) const
+{
+  // to make sure the address of jal 4-byte aligned.
+  return align_up(current_offset, alignment_required()) - current_offset;
+}
+
+// RVC: With RVC a call may get 2-byte aligned.
```

Comment:
Looks like this comment dupliates the one for CallStaticJavaDirectNode::compute_padding.

Reply:
done.